src/dispatcher/actions.ts

```diff
@@ -45,8 +45,13 @@ export enum Action {
     ViewTooltip = "view_tooltip",
 
     /**
-     *  Forces the theme to reload. No additional payload information required.
+     * Forces the theme to reload. No additional payload information required.
      */
     RecheckTheme = "recheck_theme",
+
+    /**
+     * Response to manual update check, will fire multiple times during the update check.
```

Comment:
Please reference the payload, similar to the `ViewUser` description.

It also helps to keep the documentation in the same tense:
```suggestion
     * Provide status information for an ongoing update check. Should be used with a CheckUpdatesPayload.
```